ebuku.el

```diff
@@ -853,6 +853,49 @@ If an argument is excluded, get it from `ebuku-cache-default-args'."
       (setq tags (nconc (map-elt bookmark 'tags) tags)))
     (setq ebuku-tags (sort (seq-uniq tags) 'string-collate-lessp))))
 
+(defun ebuku-append-tag-to-bookmark ()
+  "Interactive function to append tag to bookmark."
+  ;; TODO fix an issue where it moves the cursor to the top after adding
+  (interactive)
+  (ebuku-update-tags-cache)
+  (let ((tag (completing-read "Append tag? " ebuku-tags))
+	(index (ebuku--get-index-at-point))
+	(inhibit-read-only t))
+    (ebuku--tags-add index tag)
+    (ebuku-refresh)
+    (message "Added tag to bookmark")))
+
+(defun ebuku--tags-join (tags)
+  "Internal function that joins a list of tags into a comma separated string of tags."
```

Comment:
"Internal function *to* join a list ..." Also, please wrap docstrings on word breaks so each line is less than 80 characters.

Reply:
Changed the sentence a little bit so it's less than 80 characters, my linter was complaining that the first line was not a complete sentence when wrapped.

In my vim setup I usually just have a ruler for the 80 character mark that I can toggle but I'd be curious to hear how you deal with seeing if any lines are longer than that or do you have a linter that can do that?

Reply:
i use [column-enforce-mode](https://github.com/jordonbiondo/column-enforce-mode) to display any characters beyond the 80-character limit in red. :-)